include/services/OswServiceTasks.h

```diff
@@ -3,21 +3,26 @@
 class OswServiceTaskBLECompanion;
 class OswServiceTaskExample;
 class OswServiceTaskMemMonitor;
+class OswServiceTaskNotifier;
 #ifdef OSW_FEATURE_WIFI
 class OswServiceTaskWiFi;
 class OswServiceTaskWebserver;
 #endif
-namespace OswServiceAllTasks {
+namespace OswServiceAllTasks
+{
```

Comment:
Well, cool that you formatted all that, but I fear our GitHub Actions will revert it to our coding style anyways - so, why do it in the first place?

Reply:
"editor.formatOnSave": true